src/main/java/org/opentripplanner/middleware/OtpMiddlewareMain.java

```diff
@@ -87,19 +91,28 @@ public static void main(String[] args) throws IOException, InterruptedException
                 TimeUnit.MINUTES
             );
 
-            // Schedule recurring job for post-trip surveys, once every half-hour to catch recently completed trips.
-            // TODO: Determine whether this should go in some other process.
             TripSurveySenderJob tripSurveySenderJob = new TripSurveySenderJob();
             Scheduler.scheduleJob(
                 tripSurveySenderJob,
                 0,
                 30,
                 TimeUnit.MINUTES
             );
+
```

Comment:
Do we want this to start at 3:00 AM as well? I have code for that. ;)

Reply:
This one runs every half hour, so it should start right away.

Reply:
Never mind ... I've seen the other files now.